content/doc/book/pipeline/syntax.adoc

```diff
@@ -1256,11 +1256,11 @@ pipeline {
 
 Stages in Declarative Pipeline may declare a list of nested stages to be run
 within them in sequential order. Note that a stage must have one and only one of
-`steps`, `parallel`, or `stages`, the last for sequential stages. The stages
-within `stages` in a stage cannot contain further `stages` themselves,
-and can only contain `parallel` if the parent `stage` is not already
-within a higher-level `parallel`. The parent stages do allow use of
-all other functionality of a stage, including `agent`, `tools, `when`, etc.
+`steps`, `parallel`, or `stages`, the last for sequential stages. It is not
+possible to nest a `parallel` block within a `stage` directive if that `stage`
```

Comment:
Suggestion: I feel like a code example of what's not possible might be helpful here. I understand it, but someone who's new to declarative might be confused.